### PR TITLE
Renamed job detail component to job configuration and created job details page(#23m9z3e)

### PR DESCRIPTION
**Pull request by azkyakhan**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/hotwax/job-manager#contribution-guideline)

### CODE REVIEW

**Comment by azkyakhan** [comment]
orders

job-detail

File: src/router/index.ts [hunk 1]
```diff
@@ -60,6 +61,12 @@ const routes: Array<RouteRecordRaw> = [
     component: Orders,
     beforeEnter: authGuard
   },
+  {
+    path: '/job-details',
+    name: 'JobDetail',
+    component: JobDetail,
```
Comment (dt2patel) [inline src/router/index.ts:67]: make "Detail" plural
```suggestion
    component: JobDetails,
```

**Review by dt2patel — CHANGES_REQUESTED** [review]

File: src/components/JobConfiguration.vue [hunk 2]
```diff
@@ -45,6 +45,12 @@
         <ion-button size="small" fill="outline" @click="saveChanges">{{ $t("Save changes") }}</ion-button>
       </div>
     </div>
+
+    <div class="mobile-only">
```
[the remark this answers is not in the document]
Comment (azkyakhan) [reply src/components/JobConfiguration.vue:49]: If I am adding actions class here, It is not letting buttons hide on desktop view. The display flex property isn't getting overridden by display none property on desktop view.

File: src/router/index.ts (hunk 1 above)
Comment (azkyakhan) [reply src/router/index.ts:67]: Ok sir I have renamed it.

File: src/components/JobConfiguration.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (azkyakhan) [reply src/components/JobConfiguration.vue]: As per our discussion, I have added mobile-only class in the desktop view of the inline styling of the job configuration component and added actions class to mobile-only buttons.

File: src/components/JobConfiguration.vue [hunk 3]
```diff
@@ -62,6 +62,12 @@
         <ion-button size="small" fill="outline" @click="saveChanges()">{{ $t("Save changes") }}</ion-button>
       </div>
     </div>
+
+    <div class=" actions mobile-only">
+      <ion-button expand="block" fill="outline" color="medium" @click="skipJob">{{ $t("Skip once") }}</ion-button>
```
Comment (adityasharma7) [inline src/components/JobConfiguration.vue:67]: I think we should add same condition for both condition.

File: src/components/JobConfiguration.vue [hunk 4]
```diff
@@ -75,10 +81,10 @@ import {
   IonIcon,
   IonItem,
   IonLabel,
-  IonList,
```
Comment (adityasharma7) [inline src/components/JobConfiguration.vue:78]: Unwanted change

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/components/JobConfiguration.vue (hunk 3 above)
Comment (azkyakhan) [reply src/components/JobConfiguration.vue:67]: I have added conditions to buttons in the mobile view also.

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**